README.md

```diff
@@ -32,6 +32,11 @@ OpenModelica FMI &amp; TLM based simulator
 
 ### Windows
 
+The batch scripts are developed for *MS Visual Studio 14 2015 Win64*.
```

Comment:
I guess you want to have an additional line break to get a new paragraph after this line.

Reply:
I will push a modified version in a few seconds

Reply:
better?